CMakeLists.txt

```diff
@@ -1,13 +1,20 @@
 cmake_minimum_required(VERSION 3.5)
 
-project(OpenTTD)
+project(openttd)
 
 if (CMAKE_SOURCE_DIR STREQUAL CMAKE_BINARY_DIR)
     message(FATAL_ERROR "In-source builds not allowed. Please run \"cmake ..\" from the bin directory")
 endif (CMAKE_SOURCE_DIR STREQUAL CMAKE_BINARY_DIR)
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake")
 
+# Use GNUInstallDirs to allow customisation
+# but set our own default data dir
+if (NOT CMAKE_INSTALL_DATADIR)
+    set(CMAKE_INSTALL_DATADIR "share/games/openttd")
```

Comment:
This isn't quite correct; the `CMAKE_INSTALL_DATADIR` should either not be touched, should have a `CACHE` flag on it (in which case the `if` check is unnecessary), or be set to `share/games`.

e.g.

    # by default, put all the data under {PREFIX}/share/games
    set(CMAKE_INSTALL_DATADIR CACHE "share/games")

Reply:
Also, it's missing `include(GNUInstallDirs)` - so it never gets a default value set.